presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/api/PackageAPI.java

```diff
@@ -61,17 +65,20 @@
     public synchronized Response install(@NotNull @ApiParam("Url to fetch package") String locationToFetchPackage,
```

Comment:
This method should be annotated with `@Consumes({MediaType.TEXT_PLAIN})`.

The same goes the other methods in this class.